peer/network.go

```diff
@@ -260,6 +260,21 @@ func (n *network) AppRequestFailed(nodeID ids.NodeID, requestID uint32) error {
 	return handler.OnFailure(nodeID, requestID)
 }
 
+// CrossChainAppRequest is not currently implemented.
```

Comment:
can we update the comments to say that these are no-ops instead of saying that they are not implemented?

Reply:
Sure!